explore-analyze/discover/background-search.md

```diff
@@ -0,0 +1,111 @@
+* To view and restore a saved session, you must have access to **Stack Management**.
+::::
+
+:::::
+
+## Send a search to the background
+
+You can send a search to the background only after it starts running. Until then, the **Send to background** button is disabled.
+
+1. Write or edit the query.
+
+1. Select {icon}`play` **Run** (or {icon}`refresh` **Refresh** if you already ran the query at least once) to start executing the query. At this point, the {icon}`background_task` **Send to background** button becomes available.
+
+1. Select {icon}`background_task` **Send to background**. The search is sent to the background and added to the queue of background searches.
+
+You can resume your other tasks, for example start a new search, navigate to a different application, or close the browser. Once the search has completed, a notification informs you and lets you access the search to view its results.
+
+Background searches expire after 7 days. Beyond that period, you must run the search again. You can change this default value by editing the [`data.search.sessions.defaultExpiration`](kibana://reference/configuration-reference/search-sessions-settings.md) setting.
+
+## Reopen or manage background searches
+
+From the list of background searches, you can reopen and edit any searches, but also extend their validity period or delete them to keep only searches that you care about.
+
+1. Open your list of background searches using one of the following methods:
+   - Once a background search is sent to the background, a notification appears to inform you, with a link to open the list of background searches.
+   - If you miss the notification or need to access this list at any time, go to **Discover** or **Dashboards** and select the {icon}`background_task` **Background searches** button in the toolbar. This option is only available from version 9.2.
+
+     :::{tip}
+     From **Discover**, you can only view Discover background searches. And from **Dashboards**, you can only see Dashboards background searches.
+     :::
+   - Open the **Background Search** management page in {{kib}}.
+
+1. Find the background search that you want to interact with using the search or status filter options.
+   - To open it to view its results and continue your explorations, select its name. Relative dates are converted to absolute dates.
+   - To rename it, select the {icon}`boxes_horizontal` **More actions** button, then select {icon}`pencil` **Edit name**. By default, background searches get default names that indicate their execution date and time.
+   - To extend its current expiration date by another 7 days, select the {icon}`boxes_horizontal` More actions button, then select **Extend**.
+   - To delete it, select the {icon}`boxes_horizontal` More actions button, then select {icon}`trash` **Delete**.
+
+
+## Background search limitations in dashboards [_limitations]
```

Comment:
I don't see too many anchors that start with an underscore, so flagging this in case you wanted to modify it.